docs/kb/semgrep-secrets/per-product-ignore-not-working.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Product specific path ignores requires a supported CLI version
+tags: 
+    - Semgrep Secrets
+---
+
+# Why didn't Semgrep Secrets ignore the files/folders I specified?
+
+The Semgrep AppSec platform allows you to [define ignore patterns](https://semgrep.dev/docs/ignoring-files-folders-code#define-ignored-files-and-folders-in-semgrep-appsec-platform) for specific products within a project. However, these product-specific ignores for Semgrep Secrets require a CLI version `1.71.0` or later.
```

Comment:
```suggestion
The Semgrep AppSec Platform allows you to [define ignore patterns](https://semgrep.dev/docs/ignoring-files-folders-code#define-ignored-files-and-folders-in-semgrep-appsec-platform) for specific Semgrep products for each project. However, product-specific ignores for Semgrep Secrets require Semgrep version `1.71.0` or later in your CLI or CI environment.
```
Just explicitly called out that this can happen in CI environments that are using a specific docker version of Semgrep if earlier than 1.71.0.